src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c

```diff
@@ -884,29 +937,29 @@ void func_80B1C408(EnPoSisters* this, PlayState* play) {
 void EnPoSisters_Update(Actor* thisx, PlayState* play) {
     s32 pad;
     EnPoSisters* this = THIS;
-    f32 temp_f2;
+    f32 alpha;
     Vec3f checkPos;
-    s32 bgId;
+    s32 bgId; // set by BgCheck_EntityRaycastFloor5 unused by us after
```

Comment:
I don't know if this comment is needed, `bgId` being set in a BgCheck_ function and being unused is a common occurrence